protocol/x/affiliates/keeper/keeper.go

```diff
@@ -41,3 +49,154 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 }
 
 func (k Keeper) InitializeForGenesis(ctx sdk.Context) {}
+
+func (k Keeper) RegisterAffiliate(
```

Comment:
Are we going to emit this [event](https://github.com/dydxprotocol/v4-chain/blob/852995cfb181f4594a5ab61efd18137f275e82b1/proto/dydxprotocol/indexer/events/events.proto#L593) down the line? Would this be done by yourself or Jerry? Could we add a TODO time in case we forget

Reply:
I'll be handling it. have a linear task [here](https://linear.app/dydx/issue/OTE-696/emit-indexer-events-for-affilites) to track it